Assets/Plugins/Source/EOS_SDK/Generated/ContinuanceToken.cs

```diff
@@ -38,6 +38,18 @@ public Result ToString(out Utf8String outBuffer)
 			int inOutBufferLength = 1024;
 			System.IntPtr outBufferAddress = Helper.AddAllocation(inOutBufferLength);
 
+			// NOTES:
```

Comment:
Is it within our policies to be allowed to link to GitHub issues within the code? This feels like a place where that context would be helpful. Maybe a date on when this was introduced?